chromium/background-scripts/background.js

```diff
@@ -654,11 +664,21 @@ chrome.runtime.onMessage.addListener(function(message, sender, sendResponse) {
     });
   }
 
-  function storeDisabledList() {
+  function storeDisabledList(message) {
+
     const disabledListArray = Array.from(disabledList);
-    store.set({disabledList: disabledListArray}, () => {
-      sendResponse(true);
-    });
+    const httpOnceListArray = Array.from(httpOnceList);
+
+    if ('once' === message) {
```

Comment:
Is there any reason to store the session-based exception list to browser storage? A similar purposed per-session only `urlBlacklist` is also implemented without using persistent storage.

Reply:
I had it set up this way due to the fact this interfaces with the user and I needed the messaging system.

Reply:
@zoracon not sure if keeping Yoda comparisons is a good idea, confusing comparison with assignment is almost impossible in JS especially with use of `===`.